server/public/model/post_metadata.go

```diff
@@ -7,6 +7,10 @@ import (
 	"maps"
 )
 
+type BurnOnReadMetadata struct {
+	Enabled bool `json:"enabled"`
+}
```

Comment:
Do we still need this?

Reply:
nope, no needed at all, thanks, was a leftover from previous implementation.